tests/VCR/Util/HttpUtilTest.php

```diff
@@ -128,4 +128,16 @@ public function testParseHeadersMultiple()
         $outputArray = HttpUtil::parseHeaders($inputArray);
         $this->assertEquals($excpetedHeaders, $outputArray);
     }
+
+    public function testParseHeadersIncludingColons()
```

Comment:
I've checked the other tests and seems like they are nicely covering the common headers. Do you see any other edge situation which we could test?